articles/integrations/aws/session-tags.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Use AWS Session Tags with AWS APIs and Resources
+description: Learn how to use AWS Session Tags to implement role-based access control (RBAC) for AWS APIs and Resources.
+toc: true
+topics:
+  - integrations
+  - aws
+  - session-tags
+  - rbac
+contentType: how-to
+useCase:
+  - secure-an-api
+  - integrate-third-party-apps
+  - integrate-saas-sso
+---
+# Use AWS Session Tags with AWS APIs and Resources
+
+With AWS Session Tags, you can tag resources and assign users key/value pairs, which allows you to implement role-based access control (RBAC) for AWS APIs and Resources.
+
+In the example included in this guide, we will tag our AWS resources with AWS Session Tags, then create a policy for an AWS IAM role that will allow users with this role and the appropriate tags to perform specific actions on our AWS resources. We will then create a rule in Auth0 that will attach our AWS IAM role and appropriate AWS Session Tags to an Auth0 user and pass them through SAML assertions in the token. This example builds on the example provided in our [Configure Single-Sign-on (SSO) with the AWS Console](/integrations/aws/sso) guide.
+
+::: panel Amazon Web Services (AWS) Account
+Before proceeding, you will need a valid [Amazon Web Services (AWS) account](https://portal.aws.amazon.com/billing/signup#/start) for which you are an administrator.
+:::
```

Comment:
Is the stuff in the panel a Prerequisite too? Should it go below the next header?

Reply:
Done. Thank you!